code/modules/emoji/emoji_parse.dm

```diff
@@ -19,7 +19,7 @@ var/list/emojis
 			if(search)
 				emoji = lowertext(copytext(text, pos+1, search))
 				if(emoji in emojis)
-					parsed += " <img class=icon src=\ref['icons/emoji.dmi'] iconstate='[emoji]'>"
+					parsed += bicon(icon('icons/emoji.dmi', emoji))
```

Comment:
I'll do that. After this gets merged, someone can look into making all the emotes client-sided(which is possible with the browser) and allow animated emotes and what not.